### PR TITLE
Add league/commonmark

### PR DESCRIPTION
**Pull request by colinodell**
This commit adds the PHP League's CommonMark parser

### CODE REVIEW

**Comment by inverse** [comment]
:+1:

File: README.md [hunk 1]
```diff
@@ -12,23 +12,22 @@ Tested with latest stable version.
...
++----------------------+---------+------------+---------------+----------+--------------+
+| package              | version | dialect    | duration (MS) | MEM (B)  | PEAK MEM (B) |
++----------------------+---------+------------+---------------+----------+--------------+
+| erusev/parsedown     | 1.5.1   |            | 6010          | 11010048 | 11010048     |
+| cebe/markdown        | 1.0.1   |            | 7695          | 11010048 | 11010048     |
+| cebe/markdown        | 1.0.1   | gfm        | 12925         | 11272192 | 11534336     |
+| michelf/php-markdown | 1.4.1   |            | 13952         | 11272192 | 11272192     |
+| michelf/php-markdown | 1.4.1   | extra      | 19476         | 11010048 | 11272192     |
+| kzykhys/ciconia      | v1.0.2  |            | 33115         | 11796480 | 11796480     |
+| kzykhys/ciconia      | v1.0.2  | gfm        | 40104         | 12320768 | 12582912     |
```
Comment (cebe) [inline README.md:28]: how comes it uses 1.0.2 here while 1.0.3 was in the old table?

Comment (colinodell) [inline README.md:28]: Good question, I'm not really sure why Composer installed that version
instead.

**Comment by colinodell** [comment]
@kzykhys what are your thoughts on this pull request?

**Comment by colinodell** [comment]
Polite ping :)

**Comment by cebe** [comment]
FYI: I have merged this to master on my fork and added recent test results: https://github.com/cebe/Markbench#markdown-benchmarks-php

**Pull request closed without merging**